requirements.md

```diff
@@ -51,7 +51,8 @@ As we identify the requirements and constraints, a number of key contributors wi
   - [Quay][quay] - Hank Donnay <hdonnay@redhat.com>
   - [IBM Cloud Container Registry (icr)][icr]
 - Registry Vendors, Projects & Products
-  - [Docker Trusted Registry][docker-dtr]
+  - [Mirantis Secure Registry (msr, formerly Docker Trusted Registry)][msr] - Baha <bshaaban@mirantis.com>
+  - [Mirantis Container Runtime (mcr, formerly Docker Engine – Enterprise)][mcr] - Baha <bshaaban@mirantis.com>
```

Comment:
Does this belong in the registry section? Perhaps we should have another section for runtimes and admission controllers that would provide enforcement.

Reply:
The title seems generic enough, but I'll add another section for `Docker/Kubernetes/OCI Runtimes`, let me know if this title is good enough.

While I'm at it, I can also add `Mirantis Kubernetes Engine (mke, formerly Docker Enterprise/UCP)` under that section as well.

Reply:
Added a new section.

Reply:
My silence isn't me ignoring you. I'm trying to leave space for maintainers to weigh in, but I suspect they are all out this week.

Reply:
no issues :)

this can wait, it's not urgent.

Thanks for your comment though.